### ISSUE TITLE
authorize.net payment get "Credit card number is required" error message when creating invoice in Magento 2.2.6-dev

### ISSUE THREAD

**Issue by cs221313** [magento/magento2#21721]
### Preconditions (*)
1. Install Magento 2.2.6-dev CE
2. PHP 7.1.27
3. mysql 5.7.25

### Steps to reproduce (*)
1. Added a product into shopping cart.
2. Choose authorize.net payment method in checkout page.
3. Input credit card number, expired date and ccv.
4. Place order.
5. Create invoice at backend.
![image](https://user-images.githubusercontent.com/8097168/54212890-770a2600-451e-11e9-8532-1f93be458362.png)

### Expected result (*)
1. Invoice created and Money was charged.

### Actual result (*)
1. Get error message "'3(~)2(~)33(~)Credit card number is required."
![image](https://user-images.githubusercontent.com/8097168/54213184-f7308b80-451e-11e9-86e4-bcb4945a2136.png)
Following is payment.log:

[2019-03-12 15:03:37] main.DEBUG: array (
  'url' => 'https://secure.authorize.net/gateway/transact.dll',
  'request' => 
  array (
    'x_version' => 3.1,
    'x_delim_data' => 'True',
    'x_relay_response' => 'False',
    'x_test_request' => 'FALSE',
    'x_login' => '****',
    'x_tran_key' => 'XXXXXXX',
    'x_type' => 'AUTH_CAPTURE',
    'x_method' => 'CC',
    'x_invoice_num' => '000012333',
    'x_amount' => '0.01',
    'x_currency_code' => 'USD',
    'x_allow_partial_auth' => 'False',
    'x_first_name' => 'david',
    'x_last_name' => 'li',
    'x_company' => NULL,
    'x_address' => '16 Boake Street',
    'x_city' => 'Toronto',
    'x_state' => 'Armed Forces Pacific',
    'x_zip' => 'M3J 1N4',
    'x_country' => 'US',
    'x_phone' => '416666666',
    'x_fax' => NULL,
    'x_cust_id' => '19300',
    'x_customer_ip' => '127.0.0.1',
    'x_customer_tax_id' => NULL,
    'x_email' => '14569@qq.com',
    'x_email_customer' => '0',
    'x_merchant_email' => NULL,
    'x_ship_to_first_name' => 'david',
    'x_ship_to_last_name' => 'li',
    'x_ship_to_company' => NULL,
    'x_ship_to_address' => '16 Boake Street',
    'x_ship_to_city' => 'Toronto',
    'x_ship_to_state' => 'Armed Forces Pacific',
    'x_ship_to_zip' => 'M3J 1N4',
    'x_ship_to_country' => 'US',
    'x_po_num' => NULL,
    'x_tax' => '0.0000',
    'x_freight' => '0.0000',
  ),
  'response' => '3(~)2(~)33(~)Credit card number is required.(~)(~)P(~)0(~)000012333(~)(~)0.01(~)CC(~)auth_capture(~)19300(~)david(~)li(~)(~)16 Boake Street(~)Toronto(~)Armed Forces Pacific(~)M3J 1N4(~)US(~)18930630796(~)(~)1061786625@qq.com(~)david(~)li(~)(~)16 Boake Street(~)Toronto(~)Armed Forces Pacific(~)M3J 1N4(~)US(~)0.00(~)(~)0.00(~)(~)(~)A5E1E588D26077D270E15605CAAF3E69(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)(~)',
) [] []

**Comment by magento-engcom-team** [comment]
Hi @cs221313. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@cs221313 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by cs221313** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @cs221313. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @cs221313, here is your [Magento instance](https://i-21721-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21721-2-3-develop.instances.magento-community.engineering/admin](https://i-21721-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by cs221313** [comment]
In Vanilla Magento instance,  I followed the same step as I described. In step 4. I get a javascript error when click "place order" button.

Following is what I get from browser console:

AcceptCore.js:1 Uncaught TypeError: Cannot read property 'length' of null
    at b (AcceptCore.js:1)
    at Object.function.a.dispatchData.a.dispatchData (AcceptCore.js:1)
    at UiClass._createTokens (acceptjs-client.js:59)
    at UiClass.<anonymous> (acceptjs-client.js:45)
    at fire (jquery.js:3232)
    at Object.fireWith [as resolveWith] (jquery.js:3362)
    at Object.deferred.(/checkout/anonymous function) [as resolve] (https://i-21721-2-3-develop.instances.magento-community.engineering/pub/static/version1552406133/frontend/Magento/luma/en_US/jquery.js:3461:37)
    at HTMLBodyElement.<anonymous> (acceptjs-factory.js:29)
    at HTMLBodyElement.dispatch (jquery.js:5226)
    at HTMLBodyElement.elemData.handle (jquery.js:4878)

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by cs221313** [comment]
how to add label to the issue.

**Comment by ghost** [comment]
Hi @cs221313 thank you for you report, i'm not able to reproduce following steps you described, you may have wrong credentials.
![deepin-screen-recorder_Select area_20190313120435](https://user-images.githubusercontent.com/40756439/54270583-6393a500-4588-11e9-9123-fa59521c19ab.gif)

**Comment by cs221313** [comment]
Can you send me the payment.log file? So I could compare with mine.
